scripts/localize_podfile.swift

```diff
@@ -65,14 +76,18 @@ for line in lines {
         didImplicits = true
         for implicit in implicitPods {
           let implicitPodspec = repo.appendingPathComponent(implicit + ".podspec").path
-          outBuffer += "pod '\(implicit)', :path => '\(implicitPodspec)'\n"
+          outBuffer +=
+            releaseTesting
+            ? "pod '\(implicit)'\n" : "pod '\(implicit)', :path => '\(implicitPodspec)'\n"
```

Comment:
Implicit pods should not be used at all for release testing.

If all the podspecs are pushed to staging, they will be found from the `source` directive in the Podfile.

I can do a GVC tomorrow if you want to discuss verbally.

Reply:
Sure we can do the GVC. This change will set specstaging as the source of Podfile only and will not update SpecStaging repo. I am planning to update the Podfile first to let podfile find in the repo first and then we can determine how to update the SpecStaging.
I thought implict pods were involved in the tests as well since they are in the release podspec repo, but we can discuss which pods should be tested in the meeting .